config.go

```diff
@@ -81,6 +100,11 @@ func (c *Builder) mergeConfig(in map[string]string) {
 	}
 }
 
+// Err will return the last error encountered, or nil if no error has been encountered
+func (c *Builder) Err() error {
```

Comment:
Nice. I do like this idea.

Reply:
Documentation is wrong, that's meant to be "first error encountered"